JSON_Schemas/Weights_and_Condition_Scores/iscMuscleScore.json

```diff
@@ -0,0 +1,30 @@
+{
+    "required": [
+    ],
+
+    "type": "object",
+    "allOf": [
+        {
+            "$ref": "iscScoreEventCore.json"
+        },
+        {
+            "properties": {
+                "muscleScoreReference": {
+                    "type": "object",
+                    "$ref": "iscScoreReference.json",
+                    "description": "The scheme or scoring reference guideline being used"
+                },
+                "score":{
+                    "type": "string",
+                    "enum": ["A", "B", "C", "D", "E"],
+                    "description": "Value of the muscle score"
```

Comment:
Add to description - Muscle score measures the degree of muscularity of an animal and describes the shape of cattle independent of the influence of fatness.

Reply:
Added to the description